cmd/fixtures_test/scenarios/spear.json

```diff
@@ -97,11 +117,11 @@
     {
         "ID": "EXECUTE_DELAYED_SPEAR_UPDATE_RECIPE",
         "runAfter": {
-            "precondition": ["CREATE_DELAYED_SPEAR_UPDATE_RECIPE", "CREATE_SPEAR_LV1"],
+            "precondition": ["CREATE_DELAYED_SPEAR_UPDATE_RECIPE", "CREATE_SPEAR_LV1", "UPDATE_DELAYED_SPEAR_UPDATE_RECIPE"],
             "blockWait": 0
         },
         "action": "execute_recipe",
-        "paramsRef": "./recipes/spear/executions/delayed_spear_upgrader.json",
+        "paramsRef": "./recipes/spear/executions/spear_upgrader.json",
```

Comment:
why did you rename this from delayed to just spear_upgrader?

Reply:
delayed_spear_upgrade is originally created recipe.
spear_upgrader is updated recipe

Reply:
so I executed the updated recipe rather than the original recipe

Reply:
delayed_spear_upgrade execution json will become dead code if we do this,